File: API/Entities/Article.cs
using System.ComponentModel.DataAnnotations.Schema;

namespace API.Entities
{
    [Table("Articles")]
    public class Article
    {
        public int Id { get; set; }
        public string Title { get; set; }
        public string UrlIdentity { get; set; }
        public AppUser Author { get; set; }
        public int AuthorId { get; set; }
        public DateTime DateCreated { get; set; } = DateTime.UtcNow;
        public DateTime DateEdited { get; set; } = DateTime.UtcNow;
        public string Headline { get; set; }
        public string Contents { get; set; }
        public string Tags { get; set; }
        public ICollection<Comment> Comments { get; set; }
    } 
}

File: client/src/app/_models/article.ts
import { UserComment } from "./usercomment";

export interface Article {
  title: string,
  urlIdentity: string,
  authorPublicName: string,
  contents: string,
  headline: string,
  dateCreated: Date,
  dateEdited: Date,
  tags?: string[],
  comments: UserComment[]
}
